<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>233</width>
      <height>133</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Connexion&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::AutoText</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Identifiant</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_identifiant"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Mot de passe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_password">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_singin">
       <property name="styleSheet">
        <string notr="true">/* Styles par défaut du bouton */
QPushButton {
    padding: 2px 10px; /* Marge intérieure */
    border: 2px solid #3498db; /* Bordure */
    border-radius: 10px; /* Coins arrondis */
    background-color: #3498db; /* Couleur de fond */
    color: white; /* Couleur du texte */
    font-size: 16px; /* Taille de la police */
}

/* Style du bouton au survol */
QPushButton:hover {
    background-color: #2980b9; /* Couleur de fond au survol */
    border-color: #2980b9; /* Bordure au survol */
}

/* Style du bouton lorsqu'il est enfoncé */
QPushButton:pressed {
    background-color: #1f618d; /* Couleur de fond lors de l'enfoncement */
    border-color: #1f618d; /* Bordure lors de l'enfoncement */
}

/* Style du bouton lorsqu'il est désactivé */
QPushButton:disabled {
    background-color: #bdc3c7; /* Couleur de fond désactivée */
    border-color: #bdc3c7; /* Bordure désactivée */
    color: #7f8c8d; /* Couleur du texte désactivée */
}
</string>
       </property>
       <property name="text">
        <string>Se connecter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
